CONTRIBUTING.md

```diff
@@ -263,7 +263,7 @@ in the file that describes briefly why the incompatibility can be ignored.
 
 Situations when it may be acceptable to ignore a MiMa issued warning include:
 
-- if it is touching any class marked as `private[akka]`, `/** INTERNAL API*/` or similar markers
+- if it is touching any class marked as `private[pekko]`, `/** INTERNAL API*/` or similar markers
```

Comment:
does this still applied? private[org.apache.pekko]?

Reply:
Can you rephrase, I don't understand?

Reply:
Ah I understand, so technically speaking the source file does have `private[pekko]`, not `private[org.apache.pekko]` for the reasons stated earlier, so using `private[pekko]` is correct.

Reply:
Ouch, you are right.